pep-3157.txt

```diff
@@ -0,0 +1,247 @@
+PEP: 3157
+Title: Syntactic formalization of f-strings
+Version: $Revision$
+Last-Modified: $Date$
+Author: Pablo Galindo <pablogsal@python.org>,
+        Batuhan Taskaya <batuhan@python.org>,
+        Lysandros Nikolaou <lisandrosnik@gmail.com>
+Sponsor: Pablo Galindo <pablogsal@python.org>
+Discussions-To: python-dev@python.org
+Type: Standards Track
+Status: Draft
+Content-Type: text/x-rst
+Python-Version: 3.9
+Created: 15-Nov-2021
+
+
+Abstract
+========
+
+This document proposes to lift some of the restrictions originally formulated in
+:pep:`498` and to provide a formalized grammar for f-strings that can be
+integrated into parser directly. The proposed syntactic formalization of
```

Comment:
Should we rephrase it by initially emphasizing that this is about formalization (instead of lifting/relaxing the f-string grammar) and then mention that through this we can lift restrictions imposed originally because of the implementation (potentially with a link pointing towards the section the changes are covered)?

Reply:
I agree, but I don't want that it seems that we are "hiding" the new stuff. For the end users the most impactful part of this PEP is, actually, lifting the restrictions so in some way that's going to be the one we should clarify more, I think.